chain/client/src/client_actor.rs

```diff
@@ -656,7 +656,7 @@ impl Handler<Status> for ClientActor {
 
     #[perf]
     fn handle(&mut self, msg: Status, ctx: &mut Context<Self>) -> Self::Result {
-        let _span = tracing::debug_span!(target: "client", "handle", handler="Status").entered();
+        let _span = tracing::debug_span!(target: "client", "handle", handler = "Status").entered();
```

Comment:
But why?

Reply:
For consistency with the rest of the file.